spec/cache/repositories/impressions_repository_spec.rb

```diff
@@ -9,6 +9,7 @@
     let(:split_adapter) do
       SplitIoClient::SplitAdapter.new(nil, SplitIoClient::SplitConfig.new(mode: :nil), nil, nil, nil, nil, nil)
     end
+    let(:ip) { Socket.ip_address_list.detect { |intf| intf.ipv4_private? }.ip_address }
```

Comment:
What happens if ip could not be fetched? just like in config_split.rb unknown is set on that case, but I cannot see that here. @maksimf @patricioe